apparmor.d/groups/virt/virtiofsd

```diff
@@ -0,0 +1,47 @@
+# apparmor.d - Full set of apparmor profiles
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{LOCAL_SHARED_DIRS} = /var/lib/libvirt/shared
+include if exists <local/tunables/virtiofsd>
```

Comment:
This opens up the possibility to adjust local variables. Thoughts?

Reply:
Adjust by the user? no. For this you need to use [global variables](https://apparmor.pujol.io/variables/). Variables defined in a profile header are not modifiable outside.

Here, you should have something like I did in `libvirtd`: https://github.com/roddhjav/apparmor.d/blob/main/apparmor.d/groups/virt/libvirtd#L146-L150